src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -736,7 +736,10 @@ const VisitorSpaceSearchPage: FC = () => {
 		// Create a link element for each visitor space
 		const visitorSpaceLinks = visitorSpaces.map(
 			(visitorSpace: VisitorSpaceDropdownOption): ReactNode => (
-				<Link key={visitorSpace.id} href={`/zoeken?maintainer=${visitorSpace?.id}`}>
+				<Link
+					key={visitorSpace.id}
+					href={`/zoeken?${VisitorSpaceFilterId.Maintainer}=${visitorSpace?.id}`}
```

Comment:
Maybe we should rename the "id" to "slug" so it is more clear if we mean the OR id or the slug
```suggestion
					href={`/zoeken?${VisitorSpaceFilterId.Maintainer}=${visitorSpace?.slug}`}
```